### DISCUSSION TITLE
Azure/Cloud configuration and Pipelines

### DISCUSSION THREAD

**Discussion by sjorspa** [TimGeyssens/Umbraco-v8-Community-Website-StarterKit#12] [General]
I think it would be a good idea if the project has an option to enable it for cloud hosting.
IE easy option to use Azure Storage.

Also providing pipelines for common scenarios would be nice.

**Comment by TimGeyssens** [comment]
Good idea!

**Comment by TimGeyssens** [comment]
pipelines, would that be docs? Or is there a way of adding yaml (or other config) ? that can be imported into devops?

**Reply by NikRimington** [reply]
We could look to introduce different YAML file options, Shannon has a great one for projects going to Umbraco Cloud, and we can created one for "normal" azure dev ops deployment

**Reply by TimGeyssens** [reply]
sounds good!

**Reply by dawoe** [reply]
Since the project is hosted on github.. maybe also provide a github action ? I know building can be done with GH actions...don't know about deploying

**Reply by TimGeyssens** [reply]
yes good idea!
